### DISCUSSION TITLE
mTLS

### DISCUSSION THREAD

**Discussion by LiborVlasak** [JetClient/jet-client-support#115] [Q&A] [answered]
Hi,
is it possible to set JetClient to use the client certificate for TLS communication? I didn't find this option, also tried to set the client certificate for java in which IntelliJ IDAE runs. I'm not sure if I overlooked/misconfigured something or if it is not supported.

**Comment by AntonShuvaev** [comment, marked as answer]
Hi @LiborVlasak,

Currently, client certificates for TLS communication are not supported, but I plan to add this feature in the next version.

**Reply by AntonShuvaev** [reply]
I’ve added support for client certificates in version 2024.3.15
